tests/provision_test.go

```diff
@@ -40,13 +40,17 @@ func deleteAppAndPvc(appnames []string, pvcname string) {
 }
 
 func fsVolCreationTest() {
+	device := createPV(20)
+	createVg("lvmvg", device)
 	fstypes := []string{"ext4", "xfs", "btrfs"}
 	for _, fstype := range fstypes {
 		By("####### Creating the storage class : " + fstype + " #######")
 		createFstypeStorageClass(fstype)
-		By("creating and verifying PVC bound status", createAndVerifyPVC)
+		By("creating and verifying PVC bound status")
+		createAndVerifyPVC(true)
```

Comment:
what is the bool for?

Reply:
We have added some cases where PVC might not get Bound. Previously written cases all were expected to Bound. depending upon this arg. Validation will change.